react-locate-contractor/src/config.tsx

```diff
@@ -3,14 +3,14 @@ export default {
         issuer: 'https://dev-81535986.okta.com/oauth2/default',
         clientId: '0oa5se2c6gLXwYT2S5d7',
         scopes: ['openid', 'profile', 'email'],
-        redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`
-        // redirectUri: `http://localhost:3000/login/callback`
+        // redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`
+        redirectUri: `http://localhost:3000/login/callback`
     },
     widget: {
         issuer: 'https://dev-81535986.okta.com/oauth2/default',
         clientId: '0oa5se2c6gLXwYT2S5d7',
-        redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`,
-        // redirectUri: `http://localhost:3000/login/callback`,
+        // redirectUri: `https://yellow-sea-0fa5a340f.1.azurestaticapps.net/login/callback`,
+        redirectUri: `http://localhost:3000/login/callback`,
```

Comment:
looks like your redirect is pointing to localhost instead of Azure

Reply:
Changed it